### PR TITLE
make the new build system work on jenkins

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

File: config/production.json [hunk 1]
```diff
@@ -2,12 +2,14 @@
   "prettify": true,
   "fingerprint": ["css", "js"],
   "optimize": [],
```
Comment (celaus) [inline config/production.json:4]: no optimize?

File: config/production.json [hunk 2]
```diff
@@ -2,12 +2,14 @@
   "prettify": true,
```
Comment (celaus) [inline config/production.json:2]: prettify? is this necessary in production?

File: config/staging.json [hunk 3]
```diff
@@ -2,11 +2,13 @@
...
-  "description": "Download Crate. Easy installation of Crate in the cloud, docker, *nix and OSX. Works with Java, Python, Ruby, PHP, Node.js, erlang, Mono, Golang, and dbext",
-  "stable_version": "0.51.6",
-  "stable_version_date": "October 12th, 2015",
-  "testing_version": "0.52.0"
+  "settings": {
+    "site": "https://new.crate.io",
+    "id_segment": "qio2kmx6lv",
+    "id_facebook": "555437424606586",
+    "description": "Download Crate. Easy installation of Crate in the cloud, docker, *nix and OSX. Works with Java, Python, Ruby, PHP, Node.js, erlang, Mono, Golang, and dbext",
+    "stable_version": "0.51.6",
```
Comment (celaus) [inline config/staging.json]: 0.51.7? :)

File: config/production.json [hunk 4]
```diff
@@ -2,12 +2,14 @@
...
-  "stable_version": "0.51.6",
-  "stable_version_date": "October 12th, 2015",
-  "testing_version": "0.52.0"
+  "settings": {
+    "site": "https://crate.io",
+    "robots": true,
+    "id_segment": "vd4x4hv637",
+    "id_facebook": "555421237941538",
+    "description": "Download Crate. Easy installation of Crate in the cloud, docker, *nix and OSX. Works with Java, Python, Ruby, PHP, Node.js, erlang, Mono, Golang, and dbext",
+    "stable_version": "0.51.6",
```
Comment (celaus) [inline config/production.json]: 0.51.7? or is this corrected automatically?

File: config/production.json (hunk 2 above)
Comment (chaudum) [inline config/production.json:2]: yes

File: config/production.json (hunk 1 above)
Comment (chaudum) [inline config/production.json:4]: we don't wanna minify js/css

**Comment by celaus** [comment]
Ok, then ACK :)

**Pull request merged**